giskard-server/src/main/java/ai/giskard/service/dto/ml/ProjectPostDTO.java

```diff
@@ -0,0 +1,23 @@
+package ai.giskard.service.dto.ml;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.constraints.NotNull;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class ProjectPostDTO {
```

Comment:
`description` property is missing, it's used here for example: `ProjectsHome.submitNewProject`